ext/native/base/PCMain.cpp

```diff
@@ -1054,23 +1033,13 @@ int main(int argc, char *argv[]) {
 			// glsl_refresh(); // auto-reloads modified GLSL shaders once per second.
 		}
 
-#ifdef USING_EGL
-		eglSwapBuffers(g_eglDisplay, g_eglSurface);
-#else
-		if (!keys[SDLK_TAB] || t - lastT >= 1.0/60.0) {
-			SDL_GL_SwapWindow(g_Screen);
-			lastT = t;
-		}
-#endif
+		graphicsContext->SwapBuffers();
```

Comment:
This unthrottle was bad because it ignored control mapping and sat here as a strange hack. If we want that kind of unthrottle we should do it properly, until then we can use frameskip unthrottle on Linux. Though, I should of course make sure that works....